api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-# .NET Core API Reference
+# .NET Core API Reference 
 
-These documents cover the API reference for .NET Core. You can use the search function 
-on the left-hand side to search for an API or use the Table of Contents on the left to navigate.
+The .NET Core API Reference documents the types and type members available in .NET Core. You can navigate to the documentation for a particular type (a class, structure, delegate, enum, or interface) or type member (a constructor, method, property, or event) in any of three ways:
 
-If you find an issue or an inconsistency, we would love to hear about it! 
-Go ahead to [.NET Core docs repo](https://github.com/dotnet/core-docs/) On Github and leave 
-an issue to let us know.
+- By using the search function in the left-hand pane to search for the **type** by name. (Search by member name is not supported.)
 
+- By navigating to the documentation for the type by expanding the **namespace** that that type is found in in the left hand pane, and then selecting the type.
+
+- By selecting the namespace in the "Namespaces in the .NET Core Class Library" section below.  
+ 
+Each topic contains the documentation for a type and all its unique public and protected members. Inherited members are not included. To view the inherited members of a type, click on one or more of its base classes in the Inheritance section immediately after the type description. 
+
+## Namespaces ##
+
+The types in the .NET Core class library documentation are organized by namespace. A namespace is a virtual label that allows a type or type member to be uniquely identified without requiring that that type name itself be unique. For example, a class library might include two types named `People` that have different members and offer different functionality. Locating them within namespaces (such as `Application.Collections.People` and `Application.ApplicationObjects.People`) allows compilers to resolve what might otherwise be ambiguous references to `People` objects.
```

Comment:
without requiring that that type name itself be unique ->
without requiring the type name itself to be unique